[2023-11-06T09:14:19.041+0000] {processor.py:157} INFO - Started process (PID=89) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:19.041+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:14:19.044+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:14:19.043+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:19.051+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:14:19.050+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:14:19.052+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:19.081+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.043 seconds
[2023-11-06T09:14:50.062+0000] {processor.py:157} INFO - Started process (PID=143) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:50.063+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:14:50.065+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:14:50.065+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:50.068+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:14:50.067+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:14:50.068+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:14:50.084+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:15:20.920+0000] {processor.py:157} INFO - Started process (PID=198) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:20.921+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:15:20.923+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:15:20.922+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:20.925+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:15:20.924+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:15:20.925+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:20.942+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-11-06T09:15:51.379+0000] {processor.py:157} INFO - Started process (PID=253) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:51.380+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:15:51.382+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:15:51.382+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:51.384+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:15:51.384+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:15:51.385+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:15:51.402+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:16:21.795+0000] {processor.py:157} INFO - Started process (PID=308) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:21.795+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:16:21.797+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:16:21.797+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:21.799+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:16:21.799+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:16:21.800+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:21.816+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-11-06T09:16:52.112+0000] {processor.py:157} INFO - Started process (PID=363) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:52.113+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:16:52.115+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:16:52.115+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:52.118+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:16:52.117+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:16:52.118+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:16:52.137+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:20:41.280+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:20:41.281+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:20:41.284+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:20:41.284+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:20:41.288+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:20:41.287+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:20:41.288+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:20:41.317+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-11-06T09:21:11.797+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:11.799+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:21:11.801+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:21:11.801+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:11.804+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:21:11.803+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:21:11.804+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:11.823+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:21:42.009+0000] {processor.py:157} INFO - Started process (PID=152) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:42.010+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:21:42.012+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:21:42.012+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:42.015+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:21:42.014+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:21:42.015+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:21:42.034+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:22:12.226+0000] {processor.py:157} INFO - Started process (PID=207) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:12.228+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:22:12.230+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:22:12.229+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:12.232+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:22:12.232+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:22:12.233+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:12.254+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-11-06T09:22:42.391+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:42.391+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:22:42.393+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:22:42.393+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:42.396+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:22:42.395+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:22:42.396+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:22:42.414+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:23:12.668+0000] {processor.py:157} INFO - Started process (PID=318) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:12.669+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:23:12.671+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:23:12.671+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:12.673+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:23:12.673+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:23:12.673+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:12.692+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:23:42.824+0000] {processor.py:157} INFO - Started process (PID=373) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:42.825+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:23:42.827+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:23:42.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:42.830+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:23:42.829+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:23:42.830+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:23:42.849+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:24:13.006+0000] {processor.py:157} INFO - Started process (PID=428) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:13.009+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:24:13.011+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:24:13.011+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:13.014+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:24:13.013+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:24:13.014+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:13.033+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.029 seconds
[2023-11-06T09:24:43.492+0000] {processor.py:157} INFO - Started process (PID=482) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:43.493+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:24:43.495+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:24:43.495+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:43.499+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:24:43.498+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:24:43.499+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:24:43.517+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:25:13.718+0000] {processor.py:157} INFO - Started process (PID=537) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:13.720+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:25:13.722+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:25:13.721+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:13.725+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:25:13.724+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:25:13.725+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:13.743+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:25:43.862+0000] {processor.py:157} INFO - Started process (PID=593) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:43.862+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:25:43.864+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:25:43.864+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:43.867+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:25:43.866+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:25:43.868+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:25:43.886+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:26:13.948+0000] {processor.py:157} INFO - Started process (PID=649) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:13.950+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:26:13.951+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:26:13.951+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:13.954+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:26:13.953+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:26:13.954+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:13.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:26:44.086+0000] {processor.py:157} INFO - Started process (PID=704) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:44.088+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:26:44.091+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:26:44.090+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:44.094+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:26:44.093+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:26:44.094+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:26:44.113+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.030 seconds
[2023-11-06T09:27:14.304+0000] {processor.py:157} INFO - Started process (PID=759) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:14.306+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:27:14.307+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:27:14.307+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:14.310+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:27:14.309+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:27:14.310+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:14.330+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.029 seconds
[2023-11-06T09:27:44.618+0000] {processor.py:157} INFO - Started process (PID=814) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:44.618+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:27:44.621+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:27:44.620+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:44.624+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:27:44.623+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:27:44.624+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:27:44.647+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.032 seconds
[2023-11-06T09:28:15.001+0000] {processor.py:157} INFO - Started process (PID=869) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:28:15.003+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:28:15.005+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:28:15.005+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:28:15.007+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:28:15.007+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:28:15.008+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:28:15.026+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:29:15.195+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:15.197+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:29:15.198+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:15.198+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:15.202+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:15.201+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:29:15.202+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:15.225+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.033 seconds
[2023-11-06T09:29:45.696+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:45.698+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:29:45.700+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:45.700+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:45.703+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:45.702+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:29:45.703+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:45.723+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-11-06T09:29:58.765+0000] {processor.py:157} INFO - Started process (PID=145) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:58.765+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:29:58.767+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:58.767+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:58.770+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:29:58.769+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:29:58.770+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:29:58.788+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:30:10.827+0000] {processor.py:157} INFO - Started process (PID=153) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:10.828+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:30:10.830+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:10.830+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:10.836+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:10.835+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 32, in <module>
    python_callable=pipeline,
NameError: name 'pipeline' is not defined
[2023-11-06T09:30:10.836+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:10.856+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-11-06T09:30:22.099+0000] {processor.py:157} INFO - Started process (PID=201) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:22.101+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:30:22.102+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:22.102+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:22.106+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:22.104+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:30:22.106+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:22.124+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:30:52.332+0000] {processor.py:157} INFO - Started process (PID=256) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:52.332+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:30:52.334+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:52.334+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:52.337+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:30:52.336+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:30:52.337+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:30:52.355+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:31:22.620+0000] {processor.py:157} INFO - Started process (PID=311) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:22.622+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:31:22.624+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:31:22.624+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:22.627+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:31:22.626+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:31:22.627+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:22.645+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:31:43.729+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:43.730+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:31:43.732+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:31:43.731+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:43.734+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:31:43.734+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:31:43.735+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:31:43.752+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:32:13.818+0000] {processor.py:157} INFO - Started process (PID=375) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:13.819+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:32:13.821+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:32:13.821+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:13.824+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:32:13.824+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:32:13.825+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:13.843+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:32:44.138+0000] {processor.py:157} INFO - Started process (PID=430) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:44.138+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:32:44.140+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:32:44.140+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:44.143+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:32:44.142+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:32:44.143+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:32:44.162+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:33:14.405+0000] {processor.py:157} INFO - Started process (PID=485) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:14.407+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:33:14.409+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:33:14.408+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:14.412+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:33:14.411+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:33:14.412+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:14.431+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:33:44.584+0000] {processor.py:157} INFO - Started process (PID=540) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:44.586+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:33:44.588+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:33:44.588+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:44.590+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:33:44.590+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:33:44.591+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:33:44.608+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:34:14.790+0000] {processor.py:157} INFO - Started process (PID=595) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:14.791+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:34:14.793+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:34:14.792+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:14.795+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:34:14.794+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:34:14.795+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:14.813+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:34:44.916+0000] {processor.py:157} INFO - Started process (PID=650) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:44.918+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:34:44.919+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:34:44.919+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:44.922+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:34:44.921+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:34:44.922+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:34:44.939+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:35:15.061+0000] {processor.py:157} INFO - Started process (PID=706) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:15.062+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:35:15.064+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:35:15.063+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:15.066+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:35:15.066+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:35:15.066+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:15.083+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:35:45.166+0000] {processor.py:157} INFO - Started process (PID=761) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:45.167+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:35:45.168+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:35:45.168+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:45.171+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:35:45.171+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:35:45.171+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:35:45.190+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:36:15.281+0000] {processor.py:157} INFO - Started process (PID=816) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:15.282+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:36:15.284+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:36:15.283+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:15.286+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:36:15.286+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:36:15.287+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:15.304+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:36:45.452+0000] {processor.py:157} INFO - Started process (PID=871) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:45.453+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:36:45.455+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:36:45.455+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:45.458+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:36:45.457+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:36:45.458+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:36:45.475+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:37:15.669+0000] {processor.py:157} INFO - Started process (PID=926) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:15.670+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:37:15.671+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:37:15.671+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:15.674+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:37:15.673+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:37:15.674+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:15.693+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:37:45.898+0000] {processor.py:157} INFO - Started process (PID=982) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:45.899+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:37:45.901+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:37:45.901+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:45.904+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:37:45.903+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:37:45.904+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:37:45.921+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:38:15.976+0000] {processor.py:157} INFO - Started process (PID=1037) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:15.978+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:38:15.980+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:38:15.980+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:15.983+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:38:15.983+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:38:15.984+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:16.006+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.034 seconds
[2023-11-06T09:38:46.192+0000] {processor.py:157} INFO - Started process (PID=1092) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:46.193+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:38:46.194+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:38:46.194+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:46.197+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:38:46.196+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:38:46.197+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:38:46.214+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:39:16.283+0000] {processor.py:157} INFO - Started process (PID=1147) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:16.284+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:39:16.286+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:39:16.285+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:16.288+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:39:16.288+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:39:16.288+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:16.308+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:39:46.502+0000] {processor.py:157} INFO - Started process (PID=1202) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:46.502+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:39:46.504+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:39:46.504+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:46.507+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:39:46.506+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:39:46.507+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:39:46.525+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:40:16.841+0000] {processor.py:157} INFO - Started process (PID=1257) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:16.843+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:40:16.845+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:40:16.845+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:16.848+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:40:16.847+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:40:16.848+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:16.865+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:40:47.086+0000] {processor.py:157} INFO - Started process (PID=1312) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:47.087+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:40:47.089+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:40:47.089+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:47.092+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:40:47.091+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:40:47.092+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:40:47.110+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:41:17.251+0000] {processor.py:157} INFO - Started process (PID=1367) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:17.252+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:41:17.254+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:41:17.254+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:17.257+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:41:17.256+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:41:17.257+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:17.276+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-11-06T09:41:47.563+0000] {processor.py:157} INFO - Started process (PID=1422) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:47.564+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:41:47.566+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:41:47.566+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:47.569+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:41:47.568+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:41:47.569+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:41:47.585+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:42:17.692+0000] {processor.py:157} INFO - Started process (PID=1477) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:17.693+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:42:17.695+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:42:17.695+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:17.697+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:42:17.697+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:42:17.697+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:17.715+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:42:47.940+0000] {processor.py:157} INFO - Started process (PID=1531) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:47.942+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:42:47.944+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:42:47.943+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:47.946+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:42:47.945+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:42:47.946+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:42:47.963+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:43:18.072+0000] {processor.py:157} INFO - Started process (PID=1586) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:18.072+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:43:18.074+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:43:18.074+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:18.077+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:43:18.076+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:43:18.077+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:18.094+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-11-06T09:43:48.298+0000] {processor.py:157} INFO - Started process (PID=1641) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:48.301+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:43:48.303+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:43:48.303+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:48.306+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:43:48.305+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:43:48.306+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:43:48.323+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T09:44:18.587+0000] {processor.py:157} INFO - Started process (PID=1696) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:18.588+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:44:18.589+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:44:18.589+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:18.592+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:44:18.591+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:44:18.592+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:18.610+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-11-06T09:44:48.880+0000] {processor.py:157} INFO - Started process (PID=1751) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:48.881+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T09:44:48.883+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:44:48.883+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:48.886+0000] {logging_mixin.py:154} INFO - [2023-11-06T09:44:48.885+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T09:44:48.886+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T09:44:48.905+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-11-06T20:30:48.344+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T20:30:48.346+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T20:30:48.348+0000] {logging_mixin.py:154} INFO - [2023-11-06T20:30:48.348+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T20:30:48.354+0000] {logging_mixin.py:154} INFO - [2023-11-06T20:30:48.352+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T20:30:48.355+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T20:30:48.380+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-11-06T20:31:18.825+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T20:31:18.827+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T20:31:18.830+0000] {logging_mixin.py:154} INFO - [2023-11-06T20:31:18.829+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T20:31:18.834+0000] {logging_mixin.py:154} INFO - [2023-11-06T20:31:18.833+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T20:31:18.834+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T20:31:18.853+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-11-06T22:00:54.509+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:00:54.512+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:00:54.526+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:00:54.526+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:00:54.540+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:00:54.532+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:00:54.540+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:00:54.610+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.118 seconds
[2023-11-06T22:01:24.965+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:24.967+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:01:24.970+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:01:24.970+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:24.975+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:01:24.974+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:01:24.976+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:25.027+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.067 seconds
[2023-11-06T22:01:55.349+0000] {processor.py:157} INFO - Started process (PID=152) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:55.350+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:01:55.373+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:01:55.367+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:55.437+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:01:55.434+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:01:55.441+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:01:55.593+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.262 seconds
[2023-11-06T22:02:25.802+0000] {processor.py:157} INFO - Started process (PID=207) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:25.806+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:02:25.812+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:02:25.811+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:25.823+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:02:25.820+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:02:25.824+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:25.872+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.080 seconds
[2023-11-06T22:02:56.152+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:56.153+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:02:56.158+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:02:56.157+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:56.166+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:02:56.164+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:02:56.167+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:02:56.206+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.061 seconds
[2023-11-06T22:03:26.946+0000] {processor.py:157} INFO - Started process (PID=317) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:26.951+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:03:26.956+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:03:26.955+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:26.966+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:03:26.963+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:03:26.967+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:27.021+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.086 seconds
[2023-11-06T22:03:57.296+0000] {processor.py:157} INFO - Started process (PID=372) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:57.297+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:03:57.300+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:03:57.300+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:57.305+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:03:57.303+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:03:57.305+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:03:57.339+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.051 seconds
[2023-11-06T22:04:27.739+0000] {processor.py:157} INFO - Started process (PID=427) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:27.742+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:04:27.747+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:04:27.746+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:27.751+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:04:27.750+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:04:27.752+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:27.787+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.055 seconds
[2023-11-06T22:04:57.892+0000] {processor.py:157} INFO - Started process (PID=483) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:57.894+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:04:57.900+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:04:57.899+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:57.907+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:04:57.905+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:04:57.907+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:04:57.948+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.063 seconds
[2023-11-06T22:05:28.223+0000] {processor.py:157} INFO - Started process (PID=538) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:28.225+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:05:28.231+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:05:28.230+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:28.241+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:05:28.239+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:05:28.242+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:28.279+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.064 seconds
[2023-11-06T22:05:58.328+0000] {processor.py:157} INFO - Started process (PID=593) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:58.331+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:05:58.335+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:05:58.334+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:58.341+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:05:58.339+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:05:58.342+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:05:58.382+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.063 seconds
[2023-11-06T22:06:28.498+0000] {processor.py:157} INFO - Started process (PID=648) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:28.500+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:06:28.504+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:06:28.504+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:28.509+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:06:28.508+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:06:28.509+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:28.543+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.052 seconds
[2023-11-06T22:06:58.622+0000] {processor.py:157} INFO - Started process (PID=703) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:58.626+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:06:58.631+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:06:58.631+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:58.640+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:06:58.638+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:06:58.641+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:06:58.685+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.073 seconds
[2023-11-06T22:07:28.807+0000] {processor.py:157} INFO - Started process (PID=758) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:28.808+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:07:28.812+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:07:28.811+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:28.820+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:07:28.819+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:07:28.820+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:28.879+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.078 seconds
[2023-11-06T22:07:58.978+0000] {processor.py:157} INFO - Started process (PID=813) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:58.980+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:07:58.987+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:07:58.987+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:58.998+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:07:58.995+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:07:58.998+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:07:59.046+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.078 seconds
[2023-11-06T22:08:29.772+0000] {processor.py:157} INFO - Started process (PID=869) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:08:29.774+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:08:29.777+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:08:29.776+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:08:29.783+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:08:29.781+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:08:29.783+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:08:29.816+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.050 seconds
[2023-11-06T22:09:00.498+0000] {processor.py:157} INFO - Started process (PID=924) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:00.500+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:09:00.506+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:09:00.505+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:00.516+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:09:00.514+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:09:00.517+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:00.561+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.073 seconds
[2023-11-06T22:09:30.631+0000] {processor.py:157} INFO - Started process (PID=979) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:30.636+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:09:30.642+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:09:30.641+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:30.653+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:09:30.651+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:09:30.654+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:09:30.704+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.081 seconds
[2023-11-06T22:10:01.090+0000] {processor.py:157} INFO - Started process (PID=1035) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:01.092+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:10:01.098+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:10:01.097+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:01.105+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:10:01.103+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:10:01.106+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:01.141+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.061 seconds
[2023-11-06T22:10:31.396+0000] {processor.py:157} INFO - Started process (PID=1090) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:31.400+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:10:31.406+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:10:31.405+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:31.413+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:10:31.412+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:10:31.414+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:10:31.453+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.066 seconds
[2023-11-06T22:11:01.701+0000] {processor.py:157} INFO - Started process (PID=1146) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:01.704+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:11:01.708+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:11:01.708+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:01.717+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:11:01.715+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:11:01.718+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:01.752+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.060 seconds
[2023-11-06T22:11:32.259+0000] {processor.py:157} INFO - Started process (PID=1201) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:32.265+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:11:32.270+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:11:32.269+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:32.277+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:11:32.275+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:11:32.277+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:11:32.316+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.065 seconds
[2023-11-06T22:12:02.432+0000] {processor.py:157} INFO - Started process (PID=1256) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:02.435+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:12:02.440+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:12:02.439+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:02.449+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:12:02.447+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:12:02.450+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:02.494+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.071 seconds
[2023-11-06T22:12:32.597+0000] {processor.py:157} INFO - Started process (PID=1309) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:32.603+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:12:32.610+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:12:32.609+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:32.617+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:12:32.615+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:12:32.617+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:12:32.654+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.066 seconds
[2023-11-06T22:13:02.801+0000] {processor.py:157} INFO - Started process (PID=1365) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:02.802+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:13:02.806+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:13:02.805+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:02.811+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:13:02.809+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:13:02.811+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:02.844+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.048 seconds
[2023-11-06T22:13:33.348+0000] {processor.py:157} INFO - Started process (PID=1420) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:33.352+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:13:33.357+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:13:33.356+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:33.364+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:13:33.362+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:13:33.365+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:13:33.410+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.071 seconds
[2023-11-06T22:14:03.696+0000] {processor.py:157} INFO - Started process (PID=1475) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:03.697+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:14:03.700+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:14:03.700+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:03.706+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:14:03.705+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:14:03.706+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:03.738+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.046 seconds
[2023-11-06T22:14:34.356+0000] {processor.py:157} INFO - Started process (PID=1530) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:34.358+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:14:34.361+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:14:34.361+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:34.366+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:14:34.365+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:14:34.367+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:14:34.402+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.052 seconds
[2023-11-06T22:15:04.571+0000] {processor.py:157} INFO - Started process (PID=1585) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:04.572+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:15:04.577+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:15:04.576+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:04.581+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:15:04.580+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:15:04.581+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:04.616+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.050 seconds
[2023-11-06T22:15:35.403+0000] {processor.py:157} INFO - Started process (PID=1640) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:35.407+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:15:35.412+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:15:35.411+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:35.420+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:15:35.418+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:15:35.421+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:15:35.464+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.069 seconds
[2023-11-06T22:16:05.944+0000] {processor.py:157} INFO - Started process (PID=1696) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:05.946+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:16:05.951+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:16:05.951+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:05.959+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:16:05.957+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:16:05.960+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:06.007+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.070 seconds
[2023-11-06T22:16:36.834+0000] {processor.py:157} INFO - Started process (PID=1752) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:36.838+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:16:36.842+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:16:36.842+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:36.850+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:16:36.848+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:16:36.851+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:16:36.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.069 seconds
[2023-11-06T22:17:07.202+0000] {processor.py:157} INFO - Started process (PID=1807) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:07.204+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:17:07.208+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:17:07.208+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:07.218+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:17:07.216+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:17:07.219+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:07.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.064 seconds
[2023-11-06T22:17:37.755+0000] {processor.py:157} INFO - Started process (PID=1862) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:37.759+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:17:37.762+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:17:37.762+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:37.767+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:17:37.766+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:17:37.767+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:17:37.797+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.053 seconds
[2023-11-06T22:18:08.650+0000] {processor.py:157} INFO - Started process (PID=1917) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:08.654+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:18:08.658+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:18:08.657+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:08.663+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:18:08.662+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:18:08.664+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:08.694+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.051 seconds
[2023-11-06T22:18:39.137+0000] {processor.py:157} INFO - Started process (PID=1972) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:39.138+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:18:39.141+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:18:39.141+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:39.146+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:18:39.145+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:18:39.147+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:18:39.184+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.054 seconds
[2023-11-06T22:19:10.055+0000] {processor.py:157} INFO - Started process (PID=2035) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:10.058+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:19:10.061+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:19:10.061+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:10.067+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:19:10.066+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:19:10.068+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:10.099+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.051 seconds
[2023-11-06T22:19:41.113+0000] {processor.py:157} INFO - Started process (PID=2091) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:41.114+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:19:41.117+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:19:41.117+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:41.123+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:19:41.121+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:19:41.123+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:19:41.157+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.051 seconds
[2023-11-06T22:20:12.052+0000] {processor.py:157} INFO - Started process (PID=2146) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:12.056+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:20:12.060+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:20:12.060+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:12.069+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:20:12.067+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:20:12.069+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:12.114+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.067 seconds
[2023-11-06T22:20:42.679+0000] {processor.py:157} INFO - Started process (PID=2201) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:42.681+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:20:42.690+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:20:42.688+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:42.699+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:20:42.696+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:20:42.701+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:20:42.749+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.076 seconds
[2023-11-06T22:21:13.216+0000] {processor.py:157} INFO - Started process (PID=2256) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:13.219+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:21:13.222+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:21:13.221+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:13.227+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:21:13.226+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:21:13.228+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:13.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.049 seconds
[2023-11-06T22:21:43.735+0000] {processor.py:157} INFO - Started process (PID=2311) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:43.739+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:21:43.744+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:21:43.744+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:43.753+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:21:43.751+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:21:43.754+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:21:43.805+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.081 seconds
[2023-11-06T22:22:14.200+0000] {processor.py:157} INFO - Started process (PID=2366) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:14.203+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:22:14.208+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:22:14.208+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:14.217+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:22:14.214+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:22:14.218+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:14.264+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.073 seconds
[2023-11-06T22:22:44.759+0000] {processor.py:157} INFO - Started process (PID=2421) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:44.763+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:22:44.768+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:22:44.767+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:44.776+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:22:44.774+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:22:44.777+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:22:44.818+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.067 seconds
[2023-11-06T22:23:14.937+0000] {processor.py:157} INFO - Started process (PID=2476) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:14.940+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:23:14.943+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:23:14.942+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:14.948+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:23:14.947+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:23:14.948+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:14.977+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.049 seconds
[2023-11-06T22:23:45.589+0000] {processor.py:157} INFO - Started process (PID=2531) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:45.591+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:23:45.596+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:23:45.596+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:45.604+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:23:45.602+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:23:45.605+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:23:45.643+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.061 seconds
[2023-11-06T22:24:15.961+0000] {processor.py:157} INFO - Started process (PID=2586) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:15.965+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:24:15.971+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:24:15.970+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:15.979+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:24:15.978+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:24:15.980+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:16.015+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.061 seconds
[2023-11-06T22:24:46.867+0000] {processor.py:157} INFO - Started process (PID=2641) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:46.871+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:24:46.876+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:24:46.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:46.884+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:24:46.882+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:24:46.885+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:24:47.171+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.313 seconds
[2023-11-06T22:25:17.384+0000] {processor.py:157} INFO - Started process (PID=2696) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:17.385+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:25:17.389+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:25:17.389+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:17.393+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:25:17.392+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:25:17.394+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:17.423+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.044 seconds
[2023-11-06T22:25:47.509+0000] {processor.py:157} INFO - Started process (PID=2751) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:47.511+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:25:47.516+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:25:47.515+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:47.522+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:25:47.521+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:25:47.523+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:25:47.562+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.060 seconds
[2023-11-06T22:26:17.661+0000] {processor.py:157} INFO - Started process (PID=2806) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:17.662+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:26:17.666+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:26:17.666+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:17.670+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:26:17.669+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:26:17.671+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:17.701+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.046 seconds
[2023-11-06T22:26:48.159+0000] {processor.py:157} INFO - Started process (PID=2861) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:48.161+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:26:48.165+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:26:48.165+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:48.171+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:26:48.170+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:26:48.172+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:26:48.203+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.052 seconds
[2023-11-06T22:27:18.324+0000] {processor.py:157} INFO - Started process (PID=2916) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:18.327+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:27:18.332+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:27:18.332+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:18.339+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:27:18.338+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:27:18.340+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:18.379+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.064 seconds
[2023-11-06T22:27:49.166+0000] {processor.py:157} INFO - Started process (PID=2971) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:49.170+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:27:49.175+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:27:49.175+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:49.185+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:27:49.182+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:27:49.185+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:27:49.235+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.078 seconds
[2023-11-06T22:28:20.097+0000] {processor.py:157} INFO - Started process (PID=3026) to work on /opt/airflow/dags/hw_dag.py
[2023-11-06T22:28:20.099+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-11-06T22:28:20.107+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:28:20.106+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:28:20.117+0000] {logging_mixin.py:154} INFO - [2023-11-06T22:28:20.115+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-11-06T22:28:20.120+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-11-06T22:28:20.180+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.092 seconds
